nengo_spinnaker/scripts/nengo_spinnaker_setup.py

```diff
@@ -0,0 +1,83 @@
+"""A script which helps users create their nengo_spinnaker config file."""
+
+import argparse
+
+from nengo_spinnaker.utils.paths import nengo_spinnaker_rc
+
+from rig import wizard
+
+import os
+
+
+CONFIG_TEMPLATE = """\
+### SpiNNaker system configuration.
+# File automatically generated by nengo_spinnaker_setup.
+[spinnaker_machine]
+hostname: {hostname}
+width: {width}
+height: {height}
+"""
+
+
+def generate_config_file(filename, dimensions, ip_address):
+    """Generate a new config file with the specified filename and
+    parameters.
+    """
+    try:
+        os.makedirs(os.path.dirname(filename))
+    except FileExistsError:
+        # Directory already created, good job!
+        pass
+
+    with open(filename, "w") as f:
+        f.write(CONFIG_TEMPLATE.format(
+            hostname=ip_address,
+            width=dimensions[0],
+            height=dimensions[1],
+        ))
+
+
+def main(args=None):
+    parser = argparse.ArgumentParser(
+        description="Interactive tool for creating a nengo_spinnaker "
+                    "config file.")
+
+    file_group = parser.add_mutually_exclusive_group()
+    file_group.add_argument("--system", "-s", dest="file",
+                            const="system", action="store_const",
+                            help="Create a system-wide config file.")
```

Comment:
Ah yes... I should have grocked https://github.com/project-rig/nengo_spinnaker/blob/34910e6851894445cd660693411dd49ee053baf6/nengo_spinnaker/utils/paths.py#L12 before using it. Shall I remove this for now?